iguana/launch_assetchains.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eo pipefail
+
+# You can now add delay line to pubkey.txt file
+source pubkey.txt
+overide_args="$@"
+seed_ip=`getent hosts zero.kolo.supernet.org | awk '{ print $1 }'`
+komodo_binary='./komodod'
+
+if [ -z "$delay" ]; then delay=20; fi
+
+./listassetchainparams | while read args; do
+  gen=""
+  if [ $[RANDOM % 10] == 1 ]; then
+      gen=" -gen -genproclimit=1"
```

Comment:
what is the motivation to CPU mine each 10th chain?

Reply:
To protect chains from being stuck with 0 miners. With 64 nodes each mining 1/10 chains.

Reply:
Each 20th now as most are mined now by usual miners.